### DISCUSSION TITLE
Is the Image and Container automatically updated to latest version of FGC?

### DISCUSSION THREAD

**Discussion by peterf81** [vogler/free-games-claimer#147] [General]
Hi,

I successfully deployed on synology via GUI. But I did not use the command you provided (docker run --rm -it -p 6080:6080 -v fgc:/fgc/data --pull=always ghcr.io/vogler/free-games-claimer), instead I configured volumes and ports via synology docker GUI. Everything works.

But about the parameter "--pull=always" ? Does it check if there is a new version available every time the container is run? Or what does it do ?

Or do I have to manually check every month or when claiming is not working if there is a new version of FGC ?

This is more a general docker question, but I did not find a satisfying answer elsewhere.

I'am asking because I know that from time to time, the stores make some changes and you update to new version to make it work again. So I have to update manually every time this happens? Delete old container, and create new container manually ?

**Comment by vogler** [comment]
> the parameter "--pull=always" ? Does it check if there is a new version available every time the container is run?

Yes.
I don't have a Synology NAS, so I don't know how they handle updates in their GUI.
You could also use https://github.com/containrrr/watchtower for updates of all images.
